README.md

```diff
@@ -104,6 +104,16 @@ In order to keep code clean and easily maintainable please use the following VS
 
 - [prettier]
 - [eslint]
+- [stylelint]
+   
+   Add the following lines to your settings.json
+
+    ```sh
+    "css.validate": false,
+    "less.validate": false,
+    "scss.validate": false,
+    "stylelint.validate": [ "css", "scss"],
+    ```
```

Comment:
Mulla meni hetki etsiessä näitä. Pitäisikö laittaa tuo `src/main/reactjs/.vscode/` versionhallintaan ja laittaa nämä sen alle `settings.json`iin, niin tulisi (oletan) automaattisesti käyttöön workspace settingseihin. Toki tämä olettaa sen että jengi avaa tuon `src/main/reactjs/` VSCodeen.